### ISSUE TITLE
Internal updater installs wrong image for pi3 on pi2 - boot failes

### ISSUE THREAD

**Issue by makroground** [j-a-n/raspberrymatic-addon-rmupdate#37]
I was at 3.53.30.20201024 and the Updater addons says to me, version 3.53.34.20201121 is availible for update. I clicked on the Update-Button (not insert any URL to an zip-file) and it starts to download the "*-rpi3.zip"-file, the recovery system flashes it also without any errors. Than die Pi restarts and could boot no more, because my Raspberry Pi is an Model 2 B, not a Model 3 B.

Temporary fix: I removed the sdcard and inserted it to one of my other Pi's, which is indeed a Model 3 B and could get the raspberrymatic back to life, but I need this Pi and would like to go back to Pi 2 B immediately!

How can I now downgrade and change the system back to Raspberry Pi Model 2 B?

**Comment by j-a-n** [comment]
This should not happen with the latest version. Which version of rmupdate was used to update the system?

**Comment by makroground** [comment]
The installed version is 1.14.10, so not the newest available. So in some cases it is my fault not updating all addons before updating raspberry matic image, but is there a way to persuade the recovery system or what else to flash again a "wrong" image? I need it to be an Raspberry Pi 2 B Image again 😥

I allready tried to define the URL https://github.com/jens-maus/RaspberryMatic/releases/download/3.53.34.20201121/RaspberryMatic-3.53.34.20201121-**rpi2.**zip but it ends up in recovery system, without up/-downgrading firmware. Probably the Recovery don't accept the update file, but has accepted an pi3 image on pi2... 😫

**Comment by makroground** [comment]
I solved the problem by:

1. successful booting the system on my Raspberry Pi 3 B, which the image after the wrong update was designed for
2. generated a system backup
3. (optional only for my safety) use my good old Linux PC to generate a backup image of the whole sdcard
4. download the latest release of Raspberrymatic from git - extracted it...
5. applied the .img file to the sdcard
6. inserted the sdcard into my good old Raspberry Pi 2 B 🤩 and boot
7. after the WebUI was online, I navigated straight to the system settings and imported the system backup obtained from step 2
8. a reboot later the whole Raspberrymatic was as if nothing had happened 🧙‍♂️

@j-a-n I trust in you, that the problem is really solved now in the current versions of this addon, because I really loved your work, but this was a shock to me. Don't want to go without rmupdate 💪
